Lib/test/test_free_threading/test_tokenize.py

```diff
@@ -0,0 +1,54 @@
+import io
+import time
+import unittest
+import tokenize
+from functools import partial
+from threading import Thread
+
+
+class TestTokenize(unittest.TestCase):
```

Comment:
For this to pass on WASI, you need to `from test.support import threading_helper` and decorate the class with `@threading_helper.requires_working_threading()`